public/pages/workflow_detail/workflow_inputs/processor_inputs/input_transform_modal.tsx

```diff
@@ -59,10 +66,26 @@ export function InputTransformModal(props: InputTransformModalProps) {
 
   // source input / transformed output state
   const [sourceInput, setSourceInput] = useState<string>('[]');
-  const [transformedOutput, setTransformedOutput] = useState<string>('[]');
+  const [transformedOutput, setTransformedOutput] = useState<string>('{}');
 
   // get the current input map
-  const map = getIn(values, `ingest.enrich.${props.config.id}.inputMap`);
+  const map = getIn(
+    values,
+    `ingest.enrich.${props.config.id}.inputMap`
```

Comment:
will the path always exist? should we handle the case where the path may not exist?

Reply:
It does always exist. But, I've actually simplified this since the full path is passed in as a prop already. see latest commit.